.github/workflows/community-contribution-label.yml

```diff
@@ -0,0 +1,12 @@
+name: Test Community Contribution Label
```

Comment:
@iamshobhraj if you were the one introducing this to the production repositories, I wanted to mention right away that it'd be nice to remove "Test" from the action's name - so that you don't need to make this update later on five times :)